nbconvert/exporters/latex.py

```diff
@@ -44,6 +44,10 @@ def _template_skeleton_path_default(self):
 
     output_mimetype = 'text/latex'
 
+    def default_filters(self):
+        yield from super(LatexExporter, self).default_filters()
```

Comment:
Sorry, I forgot when I gave the example that `yield from` is Python 3 only. To make this work on Python 2, we'll need to expand it:

```python
for f in super(LatexExporter, self).default_filters():
    yield f
```